assets/src/dashboard/app/serializers/stories.js

```diff
@@ -51,7 +51,7 @@ export default function reshapeStoryObject(originalStoryData) {
     lockUser: {
       id: lockUser[0].id,
       name: lockUser[0].name,
-      avatar: lockUser[0].avatar_urls['96'] || null,
+      avatar: lockUser[0].avatar_urls?.['96'] || null,
```

Comment:
Do we need the same in the editor as well?

Reply:
Nope

https://github.com/google/web-stories-wp/blob/0de09fdba2f5e96dce3d7467a5bf3568f61a8849/assets/src/edit-story/app/story/effects/useLoadStory.js#L89

https://github.com/google/web-stories-wp/blob/0de09fdba2f5e96dce3d7467a5bf3568f61a8849/assets/src/edit-story/components/postLock/postLock.js#L86

I tested avatar disabling in dashboard in original PR. I think something changed to break this. Either way, it is a simple fix.